index-test.js

```diff
@@ -748,4 +748,16 @@ describe('reactElementToJSXString(ReactElement)', () => {
   </div>
 </div>`);
   });
+
+  it('should return the functionValue value when "showFunctions" is true and functionValue is a valid string', () => {
+    expect(
+      reactElementToJSXString(<div fn={() => 'value'}/>, {showFunctions: true, functionValue: 'Replace me'})
```

Comment:
> By the time the arrow function reaches .toString, it gets transpiled to
> function functionValue(i) { return i++; }}

No big deal, this is only because you must be running some babel transformation, when running in latest nodejs with no transforms, should work.

At least you get the right function body.

Let me know when it's ready.